com.microsoft.mrtk.graphicstools.unity/Runtime/Shaders/GraphicsToolsStandard.shader

```diff
@@ -189,6 +189,30 @@ Shader "Graphics Tools/Standard"
             ENDHLSL
         }
 
+        Pass
+        {
+            Name "GT_ShadowCaster"
```

Comment:
Nit, I don't think this breaks anything? But we haven't prefixed the other passes with "GT_"

```suggestion
            Name "ShadowCaster"
```